### PR TITLE
Create leveling system

### LINKED ISSUE #38: Create leveling system
Keep track of player's exp. Add exp score per kill

User should be able to use somehow:

`getLevel()` returns player's level
`getCurrentExp()` returns player's current exp
`getMaxExp()` returns maximum amount of exp on current player's level

User should be able to attach callback when player's level increases. It should be some `System.out.println()` for testing purposes

### PR DESCRIPTION
**Pull request by karl-crl**

### CODE REVIEW

**Comment by karl-crl** [comment]
For detail check https://github.com/PaGr0m/RogueLike/issues/38

**Comment by HarmlessEvil** [comment]
> For detail check #38

Use "linked issues" in sidebar, like this:

File: src/main/java/ru/itmo/roguelike/characters/mobs/Zombie.java [hunk 1]
```diff
@@ -17,4 +20,10 @@ public Zombie(Actor target) {
         super(target);
     }
 
+    @Override
+    protected float getExp() {
+        return (float) (Math.random() * (MAX_BOUND_EXP - MIN_BOUND_EXP) + MIN_BOUND_EXP);
```
Comment (HarmlessEvil) [inline src/main/java/ru/itmo/roguelike/characters/mobs/Zombie.java]: Maybe create utility method for this?

File: src/main/java/ru/itmo/roguelike/characters/Player.java [hunk 2]
```diff
@@ -92,4 +96,26 @@ public void setCoordinate(IntCoordinate position) {
...
+
+    public void addExp(float exp) {
+        this.exp += exp;
+        System.out.println("Get exp " + this.exp);
+        if (this.exp > getMaxExp()) {
+            this.exp -= getMaxExp();
+            level++;
+            System.out.println("New level " + level);
+        }
+    }
```
Comment (HarmlessEvil) [inline src/main/java/ru/itmo/roguelike/characters/Player.java:119]: I like it 👍

File: src/main/java/ru/itmo/roguelike/characters/mobs/Zombie.java (hunk 1 above)
Comment (karl-crl) [reply src/main/java/ru/itmo/roguelike/characters/mobs/Zombie.java]: Add

**Review by HarmlessEvil — APPROVED** [review]

**Comment by karl-crl** [comment]
To be rewritted (again ... )

**Pull request closed without merging**